OrbitQt/orbitmainwindow.ui

```diff
@@ -435,6 +437,16 @@
     <string>Check false</string>
    </property>
   </action>
+  <action name="actionNPE">
+   <property name="text">
+    <string>NPE</string>
```

Comment:
What does the E stand for in NPE?  Should we display something more verbose like "nullptr dereference"?

Reply:
or "nullptr exception"?

Reply:
I meant Null Pointer Exception: sorry for Java here. Renamed it now.